core/Cargo.toml

```diff
@@ -30,7 +30,7 @@ rand = "0.8"
 uuid = { version = "1.1", features = ["v4"] }
 parking_lot = "0.12"
 axum = "0.5"
-rusqlite = { version = "0.29", features = ["bundled"] }
+rusqlite = { version = "0.29", features = ["bundled", "serde_json"] }
```

Comment:
this is really cool, gives you `ToSql` for `serde_json::Value`

Reply:
hope is fine @spolu

Reply:
Sure what's the purpose?

Reply:
I presume this is to turn arbitrary json params into sql ready variables in params_from_iter?

Which means get_insert_params return an untyped Value :/ 

Why not use a small typed enum as we did in the experiment?

Reply:
Implemented a SqlParam enum